etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy

```diff
@@ -363,18 +374,6 @@ class TreatmentObsMigrator extends ObsMigrator {
             WHERE value_datetime = '0000-00-00';
         ''')
 
-        executeMysql("Migrate ART start date from the intake form", '''
```

Comment:
I think this one is still needed— the other that you added won't pick up the PIH:13156 group.

Reply:
@cioan You disagree or...?

Reply:
I am sorry @brandones I did not understand what you are referring to. Can you please provide more context? Sorry, I should have paid more attention to this because this commit has broken the build last night.

Reply:
No worries. It's that I think we need two blocks which add the ART start date to an obs group, because there are two obs groups (PIH:13156 in migrateArtState and PIH:6116 in migrateArtPlan). You wrote that block for the PIH:6116 obs group in migrateArtPlan:

```
        executeMysql("Migrate ART start date from the intake form", '''
            INSERT INTO tmp_obs (
                obs_group_id,
                source_encounter_id, 
                concept_uuid, 
                value_datetime)
            SELECT 
                a.obs_group_id as obs_group_id, 
                a.source_encounter_id as source_encounter_id, 
                concept_uuid_from_mapping('CIEL', '159599') as concept_uuid,
                try_to_fix_date(o.comments) as value_datetime
            FROM hivmigration_tmp_arv_regimen a, hivmigration_ordered_other o 
            WHERE a.source_encounter_id = o.source_encounter_id AND o.ordered = 'arv_start_date' AND o.comments is not null;                        
        ''')
```

I think you need to add a similar block to migrateArtState in order to add ART start date to the PIH:13156 obs group.

Reply:
@brandones I have added the another block for migrating the ART start date from the followup here:
https://github.com/PIH/hivmigration/blob/master/etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy#L348

So, there are now 2 different blocks for migrating ART start date, one attached to PIH:13156 obsgroup and another attached to PIH:6116 obs group. Sorry, I did not find the migrateArtState that you are referring to above.

Reply:
Okay, looks good, thanks! Sorry, I meant migrateArtStatus.